### PR TITLE
[Torch Dialect] support aten.fake_quantize_per_tensor_affine

### PR DESCRIPTION
**Pull request by qingyunqu**

### CODE REVIEW

**Comment by qingyunqu** [comment]
@ramiro050 Do you have time to review this PR? Same to another two PRs, thanks!

File: lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp [hunk 1]
```diff
@@ -6996,6 +6996,54 @@ class DecomposeAtenLinalgNormOp : public OpRewritePattern<AtenLinalgNormOp> {
...
+    // std::nearby_int(input/scale)
+    Value round = rewriter.create<AtenRoundOp>(loc, op.getType(), divScale);
+    // std::nearby_int(input/scale) + zero_point
+    Value addZeroPoint = rewriter.create<AtenAddScalarOp>(
+        loc, op.getType(), round, op.getZeroPoint(), one);
+    // max(quant_min, std::nearby_int(input/scale) + zero_point)
+    Value max = rewriter.create<AtenMaximumOp>(
+        loc, op.getType(), addZeroPoint,
+        rewriter.create<AtenTensorIntOp>(loc, baseType, op.getQuantMin(), none,
+                                         none, falseVal));
```
Comment (ramiro050) [inline lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp:7027]: nit: use `/*varName=*/` for nones and falses

File: projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py [hunk 2]
```diff
@@ -4739,5 +4739,37 @@ def forward(self, x):
...
+
+# ==============================================================================
+
+class FakeQuantizePerTensorAffineModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+    @export
+    @annotate_args([
+        None,
+        ([4, 50], torch.float32, True)
```
Comment (ramiro050) [inline projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py:4753]: I would expect this decomposition to also work for dynamic shapes. If so, can you add a test with dynamic input?

**Review by ramiro050 — APPROVED** [review]
LGTM! I just have some small comments

File: projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py (hunk 2 above)
Comment (qingyunqu) [reply projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py:4753]: done

**Pull request merged**